src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -131,70 +97,58 @@ const JobHistory = _.flow(
     } finally {
       this.setState({ loading: false })
     }
-
-    if (this.state.newSubmissionId) {
-      await Utils.waitOneTick()
-      this.setState({ highlightNewSubmission: false })
-      await Utils.delay(animationLengthMillis)
-      this.setState({ newSubmissionId: undefined })
-    }
   }
 
   render() {
-    const { namespace, name, ajax: { Workspaces }, workspace: { workspace: { workflowCollectionName } } } = this.props
-    const { submissions, loading, aborting, newSubmissionId, highlightNewSubmission, linkToFC } = this.state
+    const { namespace, name, ajax: { Workspaces }, workspace: { workspace: { bucketName } } } = this.props
+    const { submissions, loading, aborting } = this.state
 
     return div({ style: styles.submissionsTable }, [
       submissions && !!submissions.length && h(AutoSizer, [
         ({ width, height }) => h(FlexTable, {
           width, height, rowCount: submissions.length,
           hoverHighlight: true,
-          styleRow: rowIndex => {
-            const { submissionId } = submissions[rowIndex]
-            if (newSubmissionId === submissionId) {
-              return {
-                transition: `background-color ${animationLengthMillis}ms cubic-bezier(0.33, -2, 0.74, 0.05)`,
-                backgroundColor: highlightNewSubmission ? colors.blue[5] : 'white'
-              }
-            }
-          },
           columns: [
             {
               size: { basis: 500, grow: 0 },
-              headerRenderer: () => h(HeaderCell, ['Job (click for details)']),
+              headerRenderer: () => h(HeaderCell, ['Submission (click for details)']),
               cellRenderer: ({ rowIndex }) => {
                 const {
-                  methodConfigurationNamespace, methodConfigurationName, submitter, submissionId
+                  methodConfigurationNamespace, methodConfigurationName, submitter, submissionId, workflowStatuses
                 } = submissions[rowIndex]
-                const viewInJobManager = linkToJobManager && !!workflowCollectionName
-                return h(Fragment, [
+                const { failed, running } = collapsedStatuses(workflowStatuses)
+
+                return a({
+                  style: {
+                    flex: 1, alignSelf: 'stretch', display: 'flex', flexDirection: 'column', justifyContent: 'center',
+                    margin: '0 -1rem', padding: '0 1rem',
+                    color: 'unset', fontWeight: 500,
+                    backgroundColor: Utils.cond([!!failed, colors.red[6]], [!!running, colors.blue[6]], colors.green[7])
```

Comment:
It's not super obvious that these are clickable aside from my cursor changing to a little hand. Maybe lightening it on hover?